src/css/cartpage.css

```diff
@@ -0,0 +1,65 @@
+.cart-page-container {
+	width: 70%;
+	margin: 2rem auto;
+}
+
+.grid-60-40 {
+	display: grid;
+	grid-template-columns: 60% 40%;
+}
+
+.card-horizontal {
+	max-width: 100%;
+	border: 1px solid var(--ternary-dark);
+	margin-bottom: 1rem;
+}
+
+.card-horizontal .img-container {
+	height: 10rem;
+}
+
+.card-horizontal img {
+	object-fit: contain;
+}
+
+.flex-space-between {
+	display: flex;
+	justify-content: space-between;
+}
+
+.cart-total-wrapper {
+	min-width: 0;
+}
+
+.quantity {
+	display: flex;
+	gap: 1rem;
+}
+
+.quantity-item {
+	height: 1.5rem;
+	width: 1.5rem;
+	border: 1px solid var(--ternary-dark);
+	font-size: 1rem;
+	color: var(--secondary-color);
+	background-color: inherit;
+}
+
+.quantity-item::-webkit-outer-spin-button,
+.quantity-item::-webkit-inner-spin-button {
+	-webkit-appearance: none;
+	margin: 0;
+}
+
+@media (max-width: 1024px) {
```

Comment:
I would suggest to use relative units like "em" for media queries. As, when we use pixels to declare our breakpoints, page zooming enacts a horizontal scrollbar and things don't work so great.

Reply:
thank you Manoj I'll refactor based on this